src/posts/2020-04-16-continuous-stuff-with-github-actions/index.md

```diff
@@ -0,0 +1,268 @@
+---
+id: 148
+path: /posts/continuous-stuff-with-github-actions
+title: "Continuous Stuff with Github Actions"
+author: miguel-palhas
+date: 2020-04-20
+tags:
+  - devops
+intro: >
+  Last year, I took on the task of improving the continuous process over at
+  Utrust. We weren't really happy with the amount of work that went into our
```

Comment:
Isn't it `utrust` now?

Reply:
it's Utrust now. It was UTRUST before

Reply:
I distinctly remember someone from their team correcting us and asking to use "utrust". The [website](https://utrust.com/) is inconsistent. Title says "Utrust", most sections use "utrust". 🤔

Reply:
according to Jorge, Utrust is the name, utrust is the logo